dids/build.gradle.kts

```diff
@@ -40,4 +40,7 @@ dependencies {
   testImplementation(kotlin("test"))
   testImplementation("io.ktor:ktor-client-mock:$ktor_version")
   testImplementation("org.mockito.kotlin:mockito-kotlin:5.1.0")
+
+  implementation("dnsjava:dnsjava:3.5.2")
+  implementation("io.github.m-m-m:mmm-binary:0.9.3")
```

Comment:
was the only open source z-base-32 lib I could find with an acceptable license that worked for the JVM. I tried chad but chad failed and failed again, this works.

Reply:
wow what a name `m-m-m.mmm-binary` 😆